### PR TITLE
Metadata validation

### PR DESCRIPTION
**Pull request by vanruch**
# Checklist:

- [x] Code follows the style guidelines
- [x] Documentation updated
- [x] Corner cases evaluated and handled
- [x] Automatic tests added/update
- [x] Automatic tests pass
- [x] Software runs (locally or hosted)
- [x] Sanity check passed
- [x] Self-review passed

### CODE REVIEW

**Comment by vanruch** [comment]
@spherefoundry reworked

File: src/utils/validations.js [hunk 1]
```diff
@@ -73,6 +73,17 @@ class ValidationAndCasting {
     return this;
   }
 
+  isCorrectId(valueNames) {
```
[the remark this answers is not in the document]
Comment (spherefoundry) [reply src/utils/validations.js]: Also: this method is almost identical to the isAddress method? Maybe a `isHex` method could be extracted? `isAddress`  and `isCorrectBundleId` could then just configure it. Less code, and also fewer tests.

**Comment by vanruch** [comment]
@spherefoundry  fixed
